Cargo.toml

```diff
@@ -26,7 +26,7 @@ etcetera = "0.8"
 itertools = "0.13"
 memchr = "2.7"
 nonempty = "0.10"
-nucleo-picker = "0.5"
+nucleo-picker = "0.6.0"
```

Comment:
Yes, this was intentional! Mainly I am worried about changing something upstream that I know will influence the choices here and this is just to prevent accidental upgrades.

Reply:
I missed this earlier as well but I woke up realising that "0.6.0" has exactly the same effect as "0.6.0"... https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html

Reply:
Oh crazy I didn't know that at all! I guess that makes sense, since it makes dependency resolution more likely to be satisfied. I guess it really is extra important to never release breaking changes in patch versions with this in mind...